bugbug/model.py

```diff
@@ -156,31 +165,116 @@ def get_human_readable_feature_names(self):
 
         return cleaned_feature_names
 
-    def get_important_features(self, cutoff, shap_values):
-        # Calculate the values that represent the fraction of the model output variability attributable
-        # to each feature across the whole dataset.
-        shap_sums = shap_values.sum(0)
-        abs_shap_sums = np.abs(shap_values).sum(0)
-        rel_shap_sums = abs_shap_sums / abs_shap_sums.sum()
+    def get_important_features(self, cutoff, shap_values, class_names):
```

Comment:
Since class_names are stored in self, class_names doesn't need to be a parameter of this function. You can just use self.class_names instead of class_names when needed